packages/shared/src/features/opportunity/graphql.ts

```diff
@@ -334,3 +351,17 @@ export const editOpportunityQuestionMutationOptions = () => {
     },
   };
 };
+
+export const recommendOpportunityScreeningQuestionsOptions = () => {
+  return {
+    mutationFn: async ({ id }: { id: string }) => {
+      const result = await gqlClient.request<{
+        recommendOpportunityScreeningQuestions: Opportunity['questions'];
+      }>(RECOMMEND_OPPORTUNITY_SCREENING_QUESTIONS_MUTATION, {
+        id,
+      });
+
+      return result.recommendOpportunityScreeningQuestions;
+    },
+  };
+};
```

Comment:
Why not in https://github.com/dailydotdev/apps/blob/main/packages/shared/src/features/opportunity/mutations.ts ?

Reply:
🤷 did not see it exists

Reply:
graphql is mutations and queries so was not thinking there is separate file for mutations